### ISSUE TITLE
[Question] shortcut for copying a line under sublime key maps 

### ISSUE THREAD

**Issue by cocoaaa** [jupyterlab/jupyterlab#6223]
Hi, 

My notebook key-binding is currently set to 'sublime', and I often find myself copying a line where the current cursor is placed on.  Unfortunately, sublime's shortcut `ctrl+L` to copy a line is overridden by the browser's default key-binding of 'ctrl+L' to `focus on the browser search bar`.  Is there a walk-around for this? If I can set a new shortcut to do so using Jupyterlab's 'shortcut setting` extension, that would be nice.  I wasn't able to find an appropriate 'command' and 'selector' to do so from looking at this [list](https://github.com/jupyterlab/jupyterlab/blob/af548c2674427da79d54ad5c4b69bb175463e9a0/packages/notebook-extension/src/index.ts#L69-L197)

I had come across [this discussion](https://github.com/jupyter/notebook/issues/1816) on Jupyter Notebook, but I don't think this would work in Jupyterlab since `custom.js` is not used in Jupyterlab?

Thanks!

**Comment by jasongrout** [comment]
> I had come across [this discussion](https://github.com/jupyter/notebook/issues/1816) on Jupyter Notebook, but I don't think this is applicable to Jupyterlab since `custom.js` is not used in Jupyterlab?

Right, though the codemirror code likely would be the same, since we also use codemirror.

As for a shortcut in jlab, I think you'd need to write a jlab extension to provide a copy-line command to the system, or to codemirror.

**Comment by jasongrout** [comment]
> Unfortunately, sublime's shortcut `ctrl+L` to copy a line

Cmd+L in sublime (on mac) just selects the line, not copies it. Ctrl+L "scrolls to selection", according to the menu.

It looks like the sublime keybindings in codemirror do indeed implement the select line for cmd+L (tested at https://codemirror.net/demo/sublime.html). And indeed, when I switch the keymap for a text editor in the menu, I'm seeing cmd+L highlight the current line (this is on master, which is basically 1.0a3 prerelease).

For the notebook - we don't apply the keymap setting to notebook cells. See #3992, #5331, #3885, #4511, etc. for context. In #4511, it does seem like someone worked on an extension to enable sublime keymaps for the notebook.
